.github/workflows/main-workflow.yaml

```
@@ -61,14 +61,18 @@ jobs:
         with:
           distribution: 'adopt'
           java-version: '11'
-      - uses: subosito/flutter-action@v1.5.0
+      - uses: subosito/flutter-action@v1.5.3
```

```suggestion
      - uses: subosito/flutter-action@v1
```

This will prevent to update every single minor changes

---

I did that because the version loaded wasn't the right one...I will try again to see if everything works 😄

---

Seems like it's working well 😄